featurebyte/feast/schema/registry.py

```diff
@@ -0,0 +1,29 @@
+"""
+Feast registry related schemas
+"""
+from typing import List, Optional
+
+from featurebyte.models.base import FeatureByteBaseModel, PydanticObjectId
+from featurebyte.models.feature_list import FeatureListModel
+from featurebyte.schema.common.base import BaseDocumentServiceUpdateSchema
+
+
+class FeastRegistryCreate(FeatureByteBaseModel):
+    """
+    Feast registry create schema
+    """
+
+    project_name: Optional[str]
```

Comment:
Is it important whether to provide the `project_name` parameter?

Reply:
I don't think it is critical. Just to provide an option here in case we want to override the default one.